notebooks/data-sources/oc-github-repo/time_to_merge_model.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<p>I think it would be worth mentioning the class distribution here, do all the bins have equal data points? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
@Shreyanand we are mentioning that in the `github_PR_EDA.ipynb` where we break down the output variable into equally sized buckets. Can also make a mention of it here